_manual/categories_and_tags.md

```diff
@@ -10,33 +10,41 @@ With Flyspray 1.0 tags are supported. They are just labels that can be applied t
 
 The differences between **categories** and **tags** in Flyspray 1.0 are:
 
-##Categories
+Categories
```

Comment:
why not just fix  by adding the space for `## Categories`?

Reply:
I didn't know it as working too. Thanks.
PR modified.